src/main/java/com/templateproject/api/controller/CommentController.java

```diff
@@ -0,0 +1,51 @@
+package com.templateproject.api.controller;
+
+import java.util.List;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.templateproject.api.entity.Comment;
+import com.templateproject.api.service.CommentService;
+
+@RestController
+@RequestMapping("/comments")
+public class CommentController {
+
+    private final CommentService commentService;
+
+    public CommentController(CommentService commentService) {
+        this.commentService = commentService;
+    }
+
+    @GetMapping
+    public List<Comment> getAllComments() {
+        return commentService.getAllComments();
+    }
+
+    @PostMapping
+    public Comment createComment(@RequestBody Comment comment) {
+        return commentService.createComment(comment);
+    }
+
+    @PutMapping("/{id}")
+    public Comment updateComment(@PathVariable("id") Long id, @RequestBody Comment updatedComment) {
```

Comment:
You may also only take a string content to update a comment. Else like i said before you will only get a User and a content string

Reply:
Done.